Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -2416,11 +2416,51 @@ def run_snippet_code_script_command(client: CoreClient, args: Dict) -> CommandRe
     )
 
 
+def form_powershell_command(unescaped_string: str) -> str:
+    """
+    Builds a Powershell command using prefix and a shell-escaped string.
+
+    Args:
+        unescaped_string (str): An unescaped command string.
+
+    Returns:
+        str: Prefixed and escaped command.
+    """
+    escaped_string = ''
+    for i in range(len(unescaped_string)):
+        char = unescaped_string[i]
```

Comment:
```suggestion
    for i, char in enumerate(unescaped_string):
```
just nice to have, your choice

Reply:
Done